boards/arm/fk7b0m1_vbt6/doc/index.rst

```diff
@@ -0,0 +1,179 @@
+.. _fk7b0m1_vbt6:
+
+FANKE FK7B0M1-VBT6 board V1.0
```

Comment:
Open question:

Zephyr board description model supports versioning. Hence, follow up revisions of this boards could theoretically be available under this same directory and also documentation.

Should we keep the "V1.0" mention here ?

Reply:
Hi, @erwango.

In this case, do I need to remove the "V1.0 board" and create the **revision.cmake** file with the content `board_check_revision(FORMAT MAJOR.MINOR.PATCH DEFAULT_REVISION 0.0.0)` ?

Since the 1.0 is the only version that currently exists, should I also create the **fk7b0m1_vbt6_1_0_0.conf** and **fk7b0m1_vbt6_1_0_0.overlay** as an empty file?  Or does those files only need when another version was released?

Thanks for your guidance.

Reply:
If this is the only one that exists today, I don't think you have to bother.
You can add this later on when (/if) a v1.X pops up.

Reply:
I removed the "V1.0".